examples/paex_read_write_wire.c

```diff
@@ -105,6 +107,30 @@ int main(void)
     if( err != paNoError ) goto error2;
 
     inputParameters.device = Pa_GetDefaultInputDevice(); /* default input device */
+
+#if USE_LOOPBACK_INPUT
```

Comment:
I don't think WASAPI-specific code belongs in our generic examples. Need to discuss with Phil.

Reply:
We could add comment for `USE_LOOPBACK_INPUT` that it affects WASAPI-only devices.

Reply:
Phil and I agree that the generic examples should not include wasapi specific code paths. They are meant to be simple examples. So these changes need to be removed from the PR.

If there is good reason to include this code at all, it needs to go in separate files. Current practice would be to move theses file to `test/patest_read_write_wire_wasapi_loopback.c` etc.

Reply:
I propose to have this test as it is very useful in checking WASAPI Loopback devices, so will create a new source file as proposed: `test/patest_read_write_wire_wasapi_loopback.c`

Reply:
Done.